src/core/core.controller.js

```diff
@@ -726,6 +726,20 @@ module.exports = function(Chart) {
 			return count;
 		},
 
+		getVisibleDatasetWeight: function() {
```

Comment:
Can this be in the dougnut controller? It doesn't seem like something that all controllers need